src/app/posts/components/post/post.component.html

```diff
@@ -0,0 +1,20 @@
+<article class="post">
```

Comment:
*ngIf="post"
check if post exist or use for post reference "?" mark like
```
{{post?.author}}
```

Reply:
Fixed in new PR Feature/hw3, added ngIf in the article element